src/components/DivElement.js

```diff
@@ -1,5 +1,14 @@
 // Import HTMLElement here
+import HTMLElement from './HTMLElement';
 
 // Define class here
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super();
+    this.tag = 'div';
+    this.content = content;
```

Comment:
While this works, you can also pass `'div'` and `content` directly into the `super()` call. That way, the parent constructor can do all the work of assigning `this.tag` and `this.content`. E.g.

```
constructor(content) {
  super('div', content);
}
```

Reply:
Ok. Thank you.